concent_api/core/tests/test_integration_report_computed_task.py

```diff
@@ -1849,3 +1849,98 @@ def test_provider_forces_computed_task_report_and_tries_to_receive_twice(self):
             )
 
         self._test_204_response(response_5)
+
+    def test_provider_forces_computed_task_report_and_tries_to_receive_ack_before_requestor_have_a_chance_to_respond_concent_should_return_http_204(self):
+        # Expected message exchange:
+        # Provider  -> Concent:    MessageForceReportComputedTask
+        # Concent   -> Provider:   HTTP 204
```

Comment:
202

Reply:
Well, I meant there a response for first receive() not send(), but maybe I should add 202 first.

Reply:
Please add it, we have it in other places (maybe not all, but we should).

Reply:
:+1: